spec/index.html

```diff
@@ -937,6 +962,47 @@ <h4 style="padding-bottom:0; margin-bottom:0;">Triple Production</h4>
       </p>
     </section>
 
+    <section id="tripleTerm">
+      <h4 id="tripleTerm" style="padding-bottom:0; margin-bottom:0;">Triple Terms</h4>
```

Comment:
Double defines this fragment.

```suggestion
      <h4 style="padding-bottom:0; margin-bottom:0;">Triple Terms</h4>
```suggestion